gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjureBasePlugin.java

```diff
@@ -93,6 +94,14 @@ private static void createIrTasks(
             compileIr.getProductDependencies().set(project.provider(pdepsExtension::getProductDependencies));
             compileIr.dependsOn(copyConjureSourcesTask);
             compileIr.dependsOn(extractCompilerTask);
+
+            project.getConfigurations().create(CONJURE_IR_CONFIGURATION, conf -> {
```

Comment:
You'll want to tag this configuration with some attributes so that gradle selects the right variant in cases where there are multiple outgoing configurations (i.e. the conjure project also has the Java plugin applied)

Reply:
This also means that you'll need to tag the consuming configuration with the same attributes